src/main/scala/scala/tools/refactoring/implementations/OrganizeImports.scala

```diff
@@ -545,17 +554,35 @@ abstract class OrganizeImports extends MultiStageRefactoring with TreeFactory wi
       else organizeImports(imports)
     }
 
+    private def organizeGroupedImports(stats: List[Tree])(importsOrganizer: List[Import] => List[Import])(transformer: Transformer): List[Tree] = {
+      stats.map { tree =>
+        List(tree)
+      }.foldLeft(List.empty[List[Tree]]) { (zero, elem) =>
+        elem.head match {
+          case addToProceedingImports: Import =>
+            def isZeroHeadAListOfImports(zeroHead: List[Tree]) = zeroHead.head.isInstanceOf[Import]
+            zero match {
+              case head :: tail if isZeroHeadAListOfImports(head) => (addToProceedingImports :: head) :: tail
+              case _ => elem :: zero
+            }
+          case _ => elem :: zero
+        }
+      }.reverse.map {
+        _.reverse
+      }.map {
+        case imps @ head :: _ if head.isInstanceOf[Import] =>
+          organizeImportsIfNoImportInSameLine(imps.asInstanceOf[List[Import]])(importsOrganizer)
```

Comment:
we should be safe because this list is done by `foldLeft` in previous transformation, all other elements should/must be single element lists. Generally following steps:
```
1. ListOfTrees.map(singleTree => List(singleTree))
2. mergeImportTreesInLists(listOfListsOfTrees)) // so List(List(Select), List(Import), List(Import), List(Ident) ...) is turned to List(List(Select), List(Import, Import), List(Ident),...)
3. processGroupedImports(mergedTrees)
```